samples/samples-powershell/.gitignore

```diff
@@ -0,0 +1,99 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+lerna-debug.log*
+
+# Diagnostic reports (https://nodejs.org/api/report.html)
+report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
+
+# Runtime data
+pids
```

Comment:
You shouldn't need most of these - they'll be inherited from the ones higher up. Delete this and then just add things back one by one as needed